app_dart/lib/src/request_handlers/get_release_branches.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:github/github.dart';
+
+import '../../cocoon_service.dart';
```

Comment:
Avoid importing your own dart packages (in this case, cocoon_service)

Reply:
Thanks for pointing this out, replaced.